staff/gextremera/leadtech-site-semantic-markup/form.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Form Simple</title>
+    <style>
+        .m-headset{
+            color: blue;            
+        }
+        .m-subheadset{
+            color:lightskyblue;
+            font-style: italic;
+        }
+        input{
+            width: 30%;
+        }
+            
+    </style>
+</head>
+<body>
+    <h1 class="m-headset">A Simple Form</h1>
+    <p class="m-subheadset">Form Fudamentals</p>
+    <form action="submit">                
+        <fieldset>
+            <legend>Customer Info</legend>
+            <label for="">Name: </label>
+            <input placeholder="Enter your name" type="text" tabindex="1" required autofocus><br /><br />
+            <label for="">Telephone: </label>
+            <input placeholder="Pattern 1-234-567-8910" type="tel" pattern="[0-9]{3}-[0-9]{2}-[0-9]{3}" tabindex="2" required><br /><br />
+            <label for="">Email address: </label>
+            <input placeholder="Enter Your Email Address" type="email" tabindex="3" required><br />
+        </fieldset>
+        <fieldset>
+            <legend>books</legend>
+            <input placeholder="" type="text" tabindex="4" required autofocus>
```

Comment:
Have a look [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist)  about how to add a kind-of autosuggestion system only w/ HTML5